src/components/content-tab-sync.tsx

```diff
@@ -174,17 +161,71 @@ function NoAuthSyncTab() {
 	);
 }
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 export function ContentTabSync( { selectedSite }: { selectedSite: SiteDetails } ) {
 	const { __ } = useI18n();
+	const { syncSites } = useSyncSites();
+	const [ connectedSites, setConnectedSites ] = useState< SyncSite[] >( [] );
+	const [ isSitesSyncSelectorOpen, setIsSitesSyncSelectorOpen ] = useState( false );
 	const { isAuthenticated } = useAuth();
 	if ( ! isAuthenticated ) {
 		return <NoAuthSyncTab />;
 	}
 
-	//TO DO: Implement logic for to display a different screen when some sites are connected
 	return (
-		<div className="flex flex-col gap-4">
-			<NoSyncedSites selectedSite={ selectedSite } />
+		<div className="flex flex-col gap-4 h-full">
+			{ connectedSites.length > 0 ? (
+				<div className="flex flex-col gap-4 mt-8 h-full">
+					{ connectedSites.map( ( site ) => (
+						<div key={ site.id } className="flex items-center gap-2 ml-8">
+							{ site.name }
+						</div>
+					) ) }
+					<div className="flex mt-auto gap-4 py-6 mx-8">
+						<Button
+							onClick={ () => {
+								setIsSitesSyncSelectorOpen( true );
+							} }
+							variant="secondary"
+							className={ '!text-a8c-blueberry !shadow-a8c-blueberry' }
+						>
+							{ __( 'Connect site' ) }
+						</Button>
+						<Button
+							onClick={ () => {
+								getIpcApi().openURL( 'https://wordpress.com/start/new-site' );
+							} }
+							variant="secondary"
+							className={ '!text-a8c-blueberry !shadow-a8c-blueberry' }
+						>
+							{ __( 'Create new site' ) }
+							<ArrowIcon />
+						</Button>
+					</div>
+				</div>
+			) : (
+				<SiteSyncDescription>
+					<CreateConnectSite openSitesSyncSelector={ () => setIsSitesSyncSelectorOpen( true ) } />
+				</SiteSyncDescription>
+			) }
+
+			{ isSitesSyncSelectorOpen && (
+				<SitesSyncModalSelector
+					onRequestClose={ () => setIsSitesSyncSelectorOpen( false ) }
+					syncSites={ syncSites }
+					onConnect={ ( siteId ) => {
+						const newConnectedSite = syncSites.find( ( site ) => site.id === siteId );
```

Comment:
I am wondering how the modal should behave if you connect a specific site e.g. should the connected site still display in the list of all sites that can be connected or perhaps should it indicate that the specific site is already connected?

Reply:
Connected sites should appear in the list, but be disabled. There’s a component variant for that [in Figma here](https://www.figma.com/design/RToz6tIuQ7nlZrikBte4GU/Studio?node-id=4445-356485&t=hZhkgZdgZyN7xQ9l-4).

![image](https://github.com/user-attachments/assets/97d7ed24-0579-409d-90e9-c07898bded90)

Reply:
Nice!

Reply:
Added the logic to display a label for sites that are already connected. https://github.com/Automattic/studio/pull/601/commits/83f2ac7f81e892df39fa87d3a6d3c10be7111c78

<img width="1032" alt="Screenshot 2024-10-18 at 11 41 33" src="https://github.com/user-attachments/assets/14ed7050-d389-4931-a780-5a4968282e93">